modules/index.js

```diff
@@ -0,0 +1,47 @@
+import { initializeApp } from './app.js';
+
+initializeApp();
+
+const booksLink = document.getElementById('booksLink');
+const addBookLink = document.getElementById('addBookLink');
+const contactLink = document.getElementById('contactLink');
+const bookListSection = document.getElementById('bookListSection');
+const addBookSection = document.getElementById('addBookSection');
+const contactSection = document.getElementById('contactSection');
+
+function activateLink(link) {
+  booksLink.classList.remove('active');
+  addBookLink.classList.remove('active');
+  contactLink.classList.remove('active');
+
+  link.classList.add('active');
+}
+
+function showSection(section) {
```

Comment:
- [ ] It would be better if we change the function key word to ES6 arrow syntax instead